src/routes/pig.py

```
@@ -25,13 +25,13 @@ async def create_pig(
     pig_service: PigServiceDep,
 ) -> PigResponse:
     pig = await pig_service.create_pig(scsc_global_status, current_user, body)
-    return pig_service.get_pig_response(pig)
+    return PigResponse.model_validate(pig)
 
 
 @pig_router.get("/pig/{id}")
 async def get_pig_by_id(id: int, pig_service: PigServiceDep) -> PigResponse:
```

Ruff (0.15.6)

[error] Function argument `id` is shadowing a Python builtin

(A002)